app/logic/Calendar/Invitation.ts

```diff
@@ -1,15 +1,14 @@
 
 /**
  * For an inbox item that represents a scheduling message, the type of message:
- * Accepted/Tentative/Declined responses, invitations, or cancellations.
+ * invitations, responses or cancellations.
+ * XXX values are chosen for compatibility with existing messages.
  */
 export enum Scheduling {
-  None = 0,
-  Accepted = 1,
-  Tentative = 2,
-  Declined = 3,
-  Request = 4,
-  Cancellation = 5,
+  NONE = 0,
+  REQUEST = 4,
+  REPLY = 6,
+  CANCEL = 5,
```

Comment:
Could you please keep the old values? They are much more logical.
(And they should be Capitalized, not UPPERCASE.)